packages/frontend/src/ee/pages/AiConversations.tsx

```diff
@@ -45,30 +45,36 @@ const useAiConversation = (projectUuid?: string) => {
         queryKey: ['ai-conversations', projectUuid],
         queryFn: async () => {
             return projectUuid
-                ? getAiConversations(projectUuid)
+                ? getAiAgentConversations(projectUuid)
                 : Promise.reject();
         },
         enabled: !!projectUuid,
     });
 };
 
-const getMessages = async (projectUuid: string, aiThreadUuid: string) => {
+const getAiAgentConversationMessages = async (
+    projectUuid: string,
+    aiThreadUuid: string,
+) => {
```

Comment:
What's the plan for deprecating this page in favor of the new one, seems like we're just missing the debug feature